mkosi.presets/20-final/mkosi.extra/usr/lib/systemd/mkosi-check-and-shutdown.sh

```diff
@@ -4,7 +4,7 @@
 systemctl --failed --no-legend | tee /failed-services
 
 # Check that secure boot keys were properly enrolled.
-if [[ -d /sys/firmware/efi/efivars/ ]]; then
+if ! systemd-detect-virt --container; then
```

Comment:
This doesn't work either because it now skips the secure boot and cmdline checks for QEMU boot.

Reply:
Nvm, didn't see the `!`. But the logs show that `mkosi-check-and-shutdown.sh` is failing but that doesn't seem to cause the ci itself to fail.

Reply:
> Nvm, didn't see the !. But the logs show that mkosi-check-and-shutdown.sh is failing but that doesn't seem to cause the ci itself to fail.

Which one exactly?

Reply:
It exits with non-zero code but QEMU boot doesn't fail (across all distros): https://github.com/systemd/systemd/actions/runs/5093597051/jobs/9156398153?pr=27806#step:9:8533